product_supplierinfo_for_customer/models/product_customerinfo.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import _, api, fields, models
+
+
+class ProductCustomerInfo(models.Model):
+    _inherit = "product.supplierinfo"
+    _name = "product.customerinfo"
+    _description = "Customer Pricelist"
+
+    name = fields.Many2one(string="Customer", help="Customer of this product")
```

Comment:
```suggestion
    partner_id = fields.Many2one(string="Customer", help="Customer of this product")
```
this `product.supplierinfo` field is finally renamed to `partner_id` in 15.0, I don't know why it wasn't adapted in the 15.0 migration but I think we should change it
might need migration script